oscm-deployer/templates/var.env.withoutSampleData.template

```diff
@@ -0,0 +1,144 @@
+# INTERNAL is used for authentification against the internal DB or against LDAP systems #	
+# OIDC is used for authentification against an external OIDC provider, e.g. Azure AD    #
+#########################################################################################
+AUTH_MODE=${AUTH_MODE}
+
+
+#################
+# Initial users #
+#################
+# The user name credentials of the platform administrator
+# For authentication mode INTERNAL: ADMIN_USER_ID=administrator
+# For authentication mode OIDC: ADMIN_USER_ID=<the user created in the OIDC provider>
+ADMIN_USER_ID=${ADMIN_USER_ID}
+# For authentication mode INTERNAL: ADMIN_USER_PWD=admin123
+# For authentication mode OIDC: ADMIN_USER_PWD=<the password of the user created in the OIDC provider>
+ADMIN_USER_PWD=${ADMIN_USER_PWD}
+
+
+#################
+# Mail settings #
+#################
+# The host name or IP address of your mail server
+# Set SMTP_HOST=oscm-maildev if you want to use embedded mail server mock
+SMTP_HOST=${SMTP_HOST}
+# The port of your mail server
+# Set SMTP_PORT=25 if you use embedded mail server mock
+SMTP_PORT=${SMTP_PORT}
+# The sender email address that OSCM should use
+SMTP_FROM=${SMTP_FROM}
+# The user name for your mail server if it requires authentication; if no
+# authentication is required, please set none
+SMTP_USER=${SMTP_USER}
+# The password for your mail server if it requires authentication; if no
+# authentication is required, please set none
+SMTP_PWD=${SMTP_PWD}
+# Whether your mail server requires authentication; can be true or false
+SMTP_AUTH=${SMTP_AUTH}
+# Whether to use TLS for mail server communication; can be true or false
+SMTP_TLS=${SMTP_TLS}
+# The sender email address that the Asynchronous Provisioning Platform (APP)
+# should use
+APP_ADMIN_MAIL_ADDRESS=${SMTP_FROM}
+# Don't change
+APP_TRUSTSTORE_BSS_ALIAS=1
+
+#######################
+# Common APP Settings #
+#######################
+# The platform administrator credentials.
+# Set the user id of the platform administrator here. 
+# For authentication mode INTERNAL: APP_USER_NAME=administrator
+# For authentication mode OIDC: APP_USER_NAME=<the user created in the OIDC provider> 
+APP_USER_NAME=${ADMIN_USER_ID}
+# For authentication mode INTERNAL: APP_USER_PWD=admin123
+# For authentication mode OIDC: APP_USER_PWD=<the password of the user created in the OIDC provider> 
+APP_USER_PWD=${ADMIN_USER_PWD}
+
+#######################
+# Controller settings #
+#######################
+# Organization ID for controller authentication
+# Please set to PLATFORM_OPERATOR if you do not use the sample data 
+# Please set to 959c9bf7 if you use the sample data (-e SAMPLE_DATA=true at oscm-deployer call)
+CONTROLLER_ORG_ID=${CONTROLLER_ORG_ID}
+# Numerical user key of the controller administrator
+# Please set to 1000 if you do not use the sample data 
+# Please set to 10000 if you  use the sample data (-e SAMPLE_DATA=true at oscm-deployer call)
+CONTROLLER_USER_KEY=${CONTROLLER_USER_KEY}
```

Comment:
Set fix values for the without sample data use case, e.g.
CONTROLLER_USER_KEY=1000
CONTROLLER_USER_NAME=${APP_USER_NAME}
etc.